configuration/white_labels/nc.py

```diff
@@ -1736,7 +1736,13 @@ def get_white_label(self) -> WhiteLabel:
                 "referralSource",
             ],
         },
-        "featureFlags": {"default": []},
+        "featureFlags": {
+            "default": {
+                "_label": "nc_show_211_link",
+                "text": "For more information or to learn more, please visit the link below : ",
+                "url": "https://nc211.org/",
+            },
+        },
```

Comment:
`featureFlags` should always be an array.
```suggestion
        "featureFlags": {"default": ["nc_show_211_link"]},
```